ibm/service/power/resource_ibm_pi_instance.go

```diff
@@ -260,10 +260,10 @@ func ResourceIBMPIInstance() *schema.Resource {
 				Description:   "Memory size",
 			},
 			PIInstanceDeploymentType: {
-				Type:        schema.TypeString,
-				ForceNew:    true,
-				Optional:    true,
-				Description: "Custom Deployment Type Information",
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validate.ValidateAllowedStringValues([]string{"EPIC", "VMNoStorage"}),
+				Description:  "Custom Deployment Type Information",
```

Comment:
why was ForceNew removed?

Reply:
@hkantare Reverted, must have mistakenly took it back.